lib/inquisitor.ex

```diff
@@ -106,4 +113,16 @@ defmodule Inquisitor do
     |> List.last()
     |> Mix.Utils.underscore()
   end
+
+  @doc false
+  def handle_non_integer(i) when is_integer(i), do: i
+  def handle_non_integer(i) do
```

Comment:
Yeah you're right.  I was initially concerned about handling floats and non numeric inputs.  Is it necessary to cover these cases?  If so I wasn't sure if exception handling was idiomatic elixir given the let it crash philosophy.